client/v2/src/main/test/java/org/opennms/velocloud/client/v2/VelocloudApiClientTest.java

```diff
@@ -25,12 +25,11 @@
  *     http://www.opennms.org/
  *     http://www.opennms.com/
  *******************************************************************************/
-package org.opennms.velocloud;
```

Comment:
The path of this file is wrong. Should be `src/test/java` not `src/main/test/java`.